src/main/groovy/com/bugsnag/android/gradle/BugsnagVariantOutputTask.groovy

```diff
@@ -4,6 +4,8 @@ import com.android.build.gradle.api.BaseVariant
 import com.android.build.gradle.api.BaseVariantOutput
 import groovy.xml.Namespace
 import org.gradle.api.DefaultTask
+import org.gradle.api.file.Directory
+import org.gradle.api.provider.Provider
```

Comment:
do we support gradle versions less than 4.7? If so would there be an issue with this `import` when used with gradle < 4.7?

Reply:
We support lower than 4.7. The import doesn't seem to be an issue as the mazerunner scenarios pass on CI, which uses gradle <4.7.